pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -73,9 +127,41 @@ SEED_NODES=("ade4d8bc8cbe014af6ebdf3cb7b1e9ad36f412c0@seeds.polkachu.com:23856",
 sed -i 's/seeds = ""/seeds = "'"${SEED_NODES[*]}"'"/' $HOME/.dydxprotocol/config/config.toml
 ```
 
-### Step 7: Create Service to Start Full Node
+The preceding command updates the `seeds` variable of `config.toml` with the list you provide.
+
+### Step 8: Use a snapshot as your node's initial state
+Using snapshots to restore or sync your full node's state saves time and effort. Using a snapshot avoids replaying all the blocks from genesis and does not require multiple binary versions for network upgrades. Instead, your node uses the snapshot as its initial state.
+
+To download and extract the snapshot contents to the default dydxprotocol home directory, first **change directories into ./dydxprotocol**. To change directories, run the following command:
+
+```bash
+cd $HOME/.dydxprotocol
+```
+
+Then, find a provider for your use case on the [Snapshot Service](https://docs.dydx.exchange/network/resources#snapshot-service) page.
 
-We will create a systemd service to start the full node.  This will allow the full node to start automatically on boot.
+> For example, if you are connecting to `dydx-mainnet-1`, you may use the provider [Bware Labs](https://bwarelabs.com/snapshots/dydx).
+
+Use the provider's instructions to download the snapshot into your `$HOME/.dydxprotocol` directory.
+
+> In most cases, you can run `wget <snapshot-web-address>`.
+
+From `$/HOME/.dydxprotocol`, run the following command, replacing the example value `your-snapshot-filename`:
+
+```bash
+lz4 -dc < your-snapshot-filename.tar.lz4 | tar xf -
+```
+Note that extracting the snapshot adds the `/data` folder in your current directory.
+
+**Change directories back to your $HOME directory for the rest of the procedure**. Run the following command:
```

Comment:
One thing to double check - I vaguely remember that if I tried to append the data directory to the current data directory, it does not work. I had to delete the current data directory and replace it with the snapshot data directory(check [here](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/scripts/create_full_node.sh#L134)

Reply:
So may wanna mention deletion of the data directory here if that is true